src/components/Navbar/Navbar.js

```diff
@@ -1,122 +1,227 @@
-import React, {Component} from 'react';
-import { Link } from 'react-router-dom'
-import './Navbar.css'
-import {ReactComponent as AboutUs} from '../../pics/AboutUs.svg';
-import {ReactComponent as Logo} from '../../pics/Logo.svg';
-import {ReactComponent as Team} from '../../pics/Team.svg';
-import {ReactComponent as Friends} from '../../pics/FriendsIcon.svg';
-import {ReactComponent as Invite} from '../../pics/InviteFriends.svg';
-import {ReactComponent as Settings} from '../../pics/MySettingsIcon.svg';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Navbar.css';
+import { ReactComponent as AboutUs } from '../../pics/AboutUs.svg';
```

Comment:
So these changes are from the prettier formatting. From what I saw, you personally haven't changed anything in Navbar.js

Reply:
yes they are from prettier, I will role this back to what they were

Reply:
You don't need to roll back changes. I just wanted to make sure.